### PR TITLE
adds pda paper scanning, and gives head job disks the tech web

### PR DESCRIPTION
**Pull request by azrael-x86**
## About The Pull Request

lets you uhhh scan papers i nto your notes app and also uhhh gives all heads the techweb app and also uhhh removes some role tablets because they dont really like need them anymore yeah cool awesome

## Why It's Good For The Game

coming one step closer to orange's vision of having techweb be heads only

## Changelog

:cl:
fix: papers are now scannable again after an oversight, and techweb now appears on all head pdas
del: removes round start tablets
/:cl:

### CODE REVIEW

**Comment by Mothblocks** [comment]
The title of your PR is incomprehensible, please update it to describe what is being changed

File: code/modules/jobs/job_types/research_director.dm [hunk 1]
```diff
@@ -60,7 +60,6 @@
 	suit = /obj/item/clothing/suit/toggle/labcoat
 	backpack_contents = list(
 		/obj/item/melee/baton/telescopic = 1,
-		/obj/item/modular_computer/tablet/preset/advanced/command = 1,
```
Comment (LemonInTheDark) [inline code/modules/jobs/job_types/research_director.dm:63]: Why is this not in the changelog?

File: code/modules/modular_computers/hardware/job_disk.dm [hunk 2]
```diff
@@ -68,6 +68,9 @@
 	if(disk_flags & DISK_STATUS)
 		progs_to_store += new /datum/computer_file/program/status(src)
 
+	if(disk_flags & DISK_SCI)
+		progs_to_store += new /datum/computer_file/program/science(src)
```
Comment (LemonInTheDark) [inline code/modules/modular_computers/hardware/job_disk.dm:72]: God this is depressing

Comment (magatsuchi) [reply code/modules/modular_computers/hardware/job_disk.dm:72]: constructive feedback where?

File: code/modules/jobs/job_types/research_director.dm (hunk 1 above)
Comment (magatsuchi) [reply code/modules/jobs/job_types/research_director.dm:63]: will add

File: code/modules/modular_computers/hardware/job_disk.dm (hunk 2 above)
Comment (LemonInTheDark) [reply code/modules/modular_computers/hardware/job_disk.dm:72]: I do not like disk_flags as a concept, it's this weird perversion of programs and it bugs me

File: code/modules/jobs/job_types/station_engineer.dm [hunk 3]
```diff
@@ -44,9 +44,6 @@
 
 	id_trim = /datum/id_trim/job/station_engineer
 	uniform = /obj/item/clothing/under/rank/engineering/engineer
-	backpack_contents = list(
-		/obj/item/modular_computer/tablet/preset/advanced/engineering = 1,
-		)
```
Comment (LemonInTheDark) [inline code/modules/jobs/job_types/station_engineer.dm:49]: Undocced change

File: code/modules/jobs/job_types/warden.dm [hunk 4]
```diff
@@ -54,7 +54,6 @@
 	suit_store = /obj/item/gun/energy/disabler
 	backpack_contents = list(
 		/obj/item/evidencebag = 1,
-		/obj/item/modular_computer/tablet/preset/advanced/security = 1,
```
Comment (LemonInTheDark) [inline code/modules/jobs/job_types/warden.dm:57]: see above

File: code/modules/jobs/job_types/scientist.dm [hunk 5]
```diff
@@ -41,9 +41,6 @@
 	id_trim = /datum/id_trim/job/scientist
 	uniform = /obj/item/clothing/under/rank/rnd/scientist
 	suit = /obj/item/clothing/suit/toggle/labcoat/science
-	backpack_contents = list(
-		/obj/item/modular_computer/tablet/preset/science = 1,
```
Comment (LemonInTheDark) [inline code/modules/jobs/job_types/scientist.dm:45]: see above

File: code/modules/jobs/job_types/roboticist.dm [hunk 6]
```diff
@@ -45,9 +45,6 @@
 	id_trim = /datum/id_trim/job/roboticist
 	uniform = /obj/item/clothing/under/rank/rnd/roboticist
 	suit = /obj/item/clothing/suit/toggle/labcoat/roboticist
-	backpack_contents = list(
-		/obj/item/modular_computer/tablet/preset/science = 1,
```
Comment (LemonInTheDark) [inline code/modules/jobs/job_types/roboticist.dm:49]: see above

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

**Pull request merged**